[2025-10-06T21:32:06.732+0000] {processor.py:161} INFO - Started process (PID=42) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:32:06.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:32:06.735+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:06.735+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:32:08.243+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:08.240+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:32:08.244+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:32:08.296+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 1.574 seconds
[2025-10-06T21:32:39.231+0000] {processor.py:161} INFO - Started process (PID=53) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:32:39.232+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:32:39.234+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:39.234+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:32:39.458+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:39.457+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:32:39.458+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:32:39.473+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.250 seconds
[2025-10-06T21:33:09.676+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:33:09.678+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:33:09.681+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:09.680+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:33:09.897+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:09.896+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:33:09.897+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:33:09.925+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.257 seconds
[2025-10-06T21:33:40.037+0000] {processor.py:161} INFO - Started process (PID=73) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:33:40.039+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:33:40.041+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:40.040+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:33:40.212+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:40.211+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:33:40.212+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:33:40.221+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.192 seconds
[2025-10-06T21:34:10.507+0000] {processor.py:161} INFO - Started process (PID=83) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:34:10.511+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:34:10.516+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:10.516+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:34:11.029+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:11.027+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:34:11.030+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:34:11.061+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.570 seconds
[2025-10-06T21:34:41.828+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:34:41.829+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:34:41.832+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:41.831+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:34:42.019+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:42.018+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:34:42.020+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:34:42.033+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.213 seconds
[2025-10-06T21:35:12.322+0000] {processor.py:161} INFO - Started process (PID=109) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:35:12.324+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:35:12.326+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:12.325+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:35:12.560+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:12.559+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:35:12.560+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:35:12.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.256 seconds
[2025-10-06T21:35:42.734+0000] {processor.py:161} INFO - Started process (PID=119) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:35:42.735+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:35:42.737+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:42.737+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:35:42.943+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:42.942+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:35:42.943+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:35:42.963+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.235 seconds
[2025-10-06T21:36:13.374+0000] {processor.py:161} INFO - Started process (PID=129) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:36:13.376+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:36:13.379+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:13.379+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:36:13.600+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:13.599+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:36:13.601+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:36:13.629+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.279 seconds
[2025-10-06T21:36:43.890+0000] {processor.py:161} INFO - Started process (PID=139) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:36:43.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:36:43.893+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:43.893+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:36:44.074+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:44.073+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:36:44.074+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:36:44.083+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.202 seconds
[2025-10-06T21:37:14.387+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/src/model_development.py
[2025-10-06T21:37:14.389+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/model_development.py for tasks to queue
[2025-10-06T21:37:14.392+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:37:14.392+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:37:14.598+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:37:14.597+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/src/model_development.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/src/model_development.py", line 5, in <module>
    from sklearn.compose import make_column_transformer
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T21:37:14.598+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/model_development.py
[2025-10-06T21:37:14.613+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/model_development.py took 0.236 seconds
